bot/cogs/booster.py

```diff
@@ -39,16 +40,19 @@ async def _monthly_booster(self) -> None:
             role = guild.get_role(BOOSTER_ROLE_ID)
             month = date.strftime("%B")
             member_tags = member_ids = member_error = ''
+            succes_count = error_count = 0
 
             for member in role.members:
-                try:
-                    await TatsuApi().add_score(member.id, BOOSTER_MONTHLY_EXP)
+                result = await TatsuApi().add_score(member.id, BOOSTER_MONTHLY_EXP)
+                if result is not None:
```

Comment:
Can be refactored to
```suggestion
                if result:
```